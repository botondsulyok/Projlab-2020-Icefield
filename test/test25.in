field spawn 0
entity master1 Eskimo spawn
entity master2 Eskimo spawn
entity master3 Eskimo spawn
item pist1 Pistol master1
item cart1 Cartridge master2
item light1 Light master3
stats spawn
stats master1
stats master2
stats master3
assemble master1
stats spawn
stats master1
stats master2
stats master3