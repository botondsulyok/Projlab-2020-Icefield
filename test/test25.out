id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master1, master2, master3
pullers: master1, master2, master3
masters: master1, master2, master3
neighbours: null

id: master1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1

id: master2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1

id: master3
type: eskimo
iceUnit: spawn
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light1

Game Won!
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master1, master2, master3
pullers: master1, master2, master3
masters: master1, master2, master3
neighbours: null

id: master1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1

id: master2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1

id: master3
type: eskimo
iceUnit: spawn
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light1

